index.js

```diff
@@ -29,5 +49,16 @@ module.exports = {
     });
     app.import(app.bowerDirectory + '/gridstack/dist/gridstack.css');
 
-  }
+  },
+
+  getOptions() {
+    let projectConfig = (this.project.config(process.env.EMBER_ENV) || {})['ember-gridstack'] || {};
+
+    let config = Object.assign({}, {
+      exclude: [],
+    }, projectConfig);
+    config.exclude = config.exclude || [];
```

Comment:
Is this just checking if the dev used a falsey value?

Reply:
since it's an `Array` i don't want it to be `null` by default. If I had a library, e.g., `lodash.defaults(`, I would use that ...